docs/content/guides/operations/providers/azure-provider/howto-azure-provider-wi/index.md

```diff
@@ -19,13 +19,94 @@ The Azure provider allows you to deploy and connect to Azure resources from a se
 - [az CLI](https://aka.ms/azcli)
 - [rad CLI]({{< ref "installation#step-1-install-the-rad-cli" >}})
 - [Setup a supported Kubernetes cluster]({{< ref "/guides/operations/kubernetes/overview#supported-clusters" >}})
+  - You will need the cluster's OIDC Issuer URL. [AKS Example](https://azure.github.io/azure-workload-identity/docs/installation/managed-clusters.html#azure-kubernetes-service-aks)
 - [Azure AD Workload Identity](https://azure.github.io/azure-workload-identity/docs/installation.html) installed in your cluster, including the [Mutating Admission Webhook](https://azure.github.io/azure-workload-identity/docs/installation/mutating-admission-webhook.html)
--	Create an app registration at Microsoft Entra ID 
--	Configure the federated credential for Radius components to deploy Azure resources following the script [here](insertscript) or manually configure the federated credential following the steps [here](https://learn.microsoft.com/en-us/entra/workload-id/workload-identity-federation-create-trust?pivots=identity-wif-apps-methods-azp#kubernetes) 
-   | Cluster Issuer URL | Service account name | Namespace | name |
-   |---------------------|----------------------|-----------|------|
-   |  |  |  |  |
-
+- [Entra ID Application Registration with federated credentials]()
+
+## Setup
+
+### install-radius-azwi.sh
```

Comment:
@willdavsmith - Can you add some details on what the script does? If they do not wish use the script, what config they need to manually